core/shared/src/main/scala/zio/Proxy.scala

```diff
@@ -0,0 +1,3 @@
+package zio
+
+object Proxy extends ProxyVersionSpecific
```

Comment:
There will probably be a better place than `zio.Proxy`.

Reply:
Let's call it `zio.ServiceProxy` because it's specifically a type of proxy for service interfaces (also the name `Proxy` is so generic, it will lead to clashes in other name spaces).